public/app/plugins/datasource/testdata/types.ts

```diff
@@ -1,5 +1,13 @@
 import { DataQuery } from '@grafana/data';
 
+import { TestDataDataSource } from './datasource';
+
+export interface QueryTypeEditorProps {
```

Comment:
This interface is not being used anywhere.

Reply:
thanks, good catch. i started to do a bigger refactor which i chickened out of! 😂